.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor.tabSize": 2,
```

Comment:
Is this file supposed to be checked in?

Reply:
yes it gets everyone's vscode in a state where they don't need to make their own workspace configs to get to the correct state (e.g. tab size 2 instead of 4)